src/cljs/rems/flash_message.cljs

```diff
@@ -19,8 +19,7 @@
    ;; On Chrome, focusing the element scrolls it fully into view,
    ;; but on Firefox the element is hidden behind the navigation menu,
    ;; so explicit scrolling is needed.
```

Comment:
this bug now happens again in responsive mode. I'll have a look if I can fix it...

Reply:
created #1578

Reply:
created #1578